replicatedhq/ship
# GitHub assets should respect render root (fixes #814)

What I Did
------------

Ensure github assets respect the render root (fixes #814)

Also, fix a null `planner.Local` when using `withStatusReceiver` [here](https://github.com/replicatedhq/ship/pull/827/files#diff-b9011e89fa3074e9c3fc633d2dc9cd6bR70)

How I Did it
------------

Pass down `renderRoot` from `CLIPlanner#build` down into the `github/render.go`, use it instead of `constants.InstallerPrefixPath`

How to verify it
------------

See example in #814

Description for the Changelog
------------

Ensure github assets respect the render root (fixes #814)
